src/connectStoreController.ts

```diff
@@ -44,40 +38,36 @@ export default function (
   })
 }
 
-function retryFetch (limit: (fn: () => PromiseLike<object>) => Promise<object>, url: string, body: object): Promise<object> { // tslint:disable-line
+function fetch(limit: (fn: () => PromiseLike<object>) => Promise<object>, url: string, body: object): Promise<object> { // tslint:disable-line
   return limit(async () => {
     const response = await got(url, {
       body: JSON.stringify(body),
-      headers: { 'Content-Type': 'application/json' },
+      headers: {'Content-Type': 'application/json'},
       method: 'POST',
+      retries: 1000,  // picking a very large number to be sure it keeps retrying as long as needed
```

Comment:
I am not sure this is a good idea because, with each retry, the delay is increased. So eventually it will just hang up. It can be considered a silent error

Reply:
I'll pass a function with a fixed delay instead